source/AI.cpp

```diff
@@ -1236,8 +1236,8 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 				{
 					auto ptr = it->shared_from_this();
 					// Scan friendly ships that are as-yet unscanned by this ship's government.
-					if((!cargoScan || Has(gov, ptr, ShipEvent::SCAN_CARGO))
-							&& (!outfitScan || Has(gov, ptr, ShipEvent::SCAN_OUTFITS)))
+					if((!(cargoScan && !it->CargoScanCompletedBy(gov)) || Has(gov, ptr, ShipEvent::SCAN_CARGO))
```

Comment:
```suggestion
					if(((!cargoScan && it->CargoScanCompletedBy(gov)) || Has(gov, ptr, ShipEvent::SCAN_CARGO))
```
simpler to understand later on

Reply:
I think this breaks the logic.  They're not equivalent from what I can tell.

Reply:
!(1 && !0) == (!1 && 0)
!(0 && !1) == (!0 && 1)
No they are the same